3rdparty/crypto/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ add_library(crypto sha256_avx2.cpp
                    sha256_sse4.cpp
                    sha256_sse41.cpp
```

Comment:
The SHA256 implementation comes with several optimised variants with significantly better performance. Before merging this PR we should check what the performance difference is when using this un-optimized SHA3 implementation. If the difference is significant, we should either hold off on this PR, or find similarly optimised implementations as SHA256.